rand_chacha/build.rs

```diff
@@ -0,0 +1,8 @@
+extern crate rustc_version;
```

Comment:
Do we need a build script?

Reply:
Yes, it is necessary for the conditional compilation of `ChaChaRng::get_word_pos` and `ChaChaRng::set_word_pos`. The alternative would be removing those or using features (which was what we did in `rand` until recently).